src/parser/jobs/mnk/modules/Brotherhood.tsx

```diff
@@ -44,7 +44,7 @@ const BROTHERHOOD_DRIFT_TOLERANCE = 1500
  * said windows to provided RoF windows.
  */
 export class Brotherhood extends RaidBuffWindow {
-	static override handle = 'brotherhood'
+	static override handle = 'Brotherhood'
```

Comment:
Rather than capitalizing the handle, we should be setting a title here. That allows it to be translated for other languages as well. Should be able to follow examples from other modules!